.github/CONTRIBUTING.md

```diff
@@ -1,28 +1,34 @@
-# Contributing to OctoberCMS
+# Contributing to October
```

Comment:
What's the reason for the name change from OctoberCMS to October?

Reply:
@LukeTowers I found the README mentioned "October" more - there's only one reference to OctoberCMS, so I followed suit. Happy to change it back if you want?

Reply:
Let's have it OctoberCMS here. I think it flows better in this document, especially for sentences like "the OctoberCMS project" vs "the October project"

Reply:
Done :)